mozdownload/scraper.py

```diff
@@ -657,8 +657,7 @@ def get_build_info(self):
         "Defines additional build information"
 
         # Internally we access builds via index
-        self.builds, self.build_index = self.get_build_info_for_version(
-            self.version)
+        self.builds, self.build_index = self.get_build_info_for_version()
```

Comment:
`self.version` is not coming through as function parameter but it is a class member. So I don't see why we can remove it here, especially when the underlying method expects an explicit version number.